bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
+
+    @classmethod
+    def setup_dependencies(cls):
+        if not cls._nltk_data_downloaded:
+            nltk.download([
+                'punkt',
+                'maxent_treebank_pos_tagger',
+                'averaged_perceptron_tagger',
+            ])
+            cls._nltk_data_downloaded = True
```

Comment:
Note that as a `GlobalBear`, this bear is typically only run once per `coala` invocation, so this variable is going to be especially useful normally, but ... with tests ... it is rather helpful to reduce spamming the logs.